src/main/java/com/nvidia/spark/rapids/jni/GpuListSliceUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.jni;
+
+import ai.rapids.cudf.*;
+
+public class GpuListSliceUtils {
+    static{
+        NativeDepsLoader.loadNativeDeps();
+    }
+
+    public static ColumnVector listSlice(ColumnView cv, int start, int length) {
```

Comment:
Missing type check for the input `cv`. Since the native try to cast it to a list column directly.

Something like:
`assert type.equals(DType.LIST) : "column type must be a LIST";`

Reply:
Also comment is preferred for these APIs.

Reply:
The cpp code is checking the type, only need to add comments.
Details:

src/main/cpp/src/ListSliceJni.cpp:

```cpp
cudf::lists_column_view lcv{*reinterpret_cast<cudf::column_view const*>(input_column)};
```

```cpp
lists_column_view::lists_column_view(column_view const& lists_column) : column_view(lists_column)
{
  CUDF_EXPECTS(type().id() == type_id::LIST, "lists_column_view only supports lists");
}
```